config/routes.rb

```diff
@@ -0,0 +1,28 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'tasks#index'
+
+  # resources :tasks do
+  #
+  #   member do
+  #     patch :complete
+  #   end
+  #
+  # end
+
+  get '/tasks', to: 'tasks#index'
+
+  get '/tasks/new', to: 'tasks#new', as: 'new_task'
+  post '/tasks', to: 'tasks#create'
+
+  get '/tasks/:id/edit', to: 'tasks#edit', as: 'edit_task'
+
+
+  patch '/tasks/:id', to: 'tasks#update', as: 'update_task'
+
+  get '/tasks/:id', to: 'tasks#show', as: 'task'
+
+  delete '/tasks/:id', to: 'tasks#destroy'
+
+  patch '/tasks/:id', to: 'task#complete', as: 'complete_task'
```

Comment:
Look here, how does this route differ from the `tasks#update` route?

You need to do something to the path to differentiate it.  Maybe:
```ruby
  patch '/tasks/:id/complete', to: 'tasks#complete', as: 'complete_task'
```

And move it up in the order.

PS:  Look at the pluralization of what you have in this route.